packages/core/src/blocks/ListItemBlockContent/CheckListItemBlockContent/CheckListItemBlockContent.ts

```diff
@@ -232,6 +228,13 @@ const checkListItemBlockContent = createStronglyTypedTiptapNode({
         this.options.domAttributes?.inlineContent || {}
       );
 
+      if (typeof getPos !== "boolean") {
+        const label =
+          "label-" + this.editor.state.doc.resolve(getPos()).node().attrs.id;
```

Comment:
can you comment (in code) or fix, why we can't just use node.attrs.id? (but need resolve + getpos)